.github/workflows/autoupdate.yml

```diff
@@ -0,0 +1,50 @@
+name: Update
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  update:
+    strategy:
+      matrix:
+        project:
```

Comment:
Probably this list is relatively static, but it still adds some maintenance burden. Have you considered adding this as a downstream cron-job instead? Maybe this can also avoid the need for a PAT?